ui/components/onboarding/Tier.tsx

```diff
@@ -23,23 +29,27 @@ export default function Tier({
   price,
   onClick,
   hasCitizen = false,
+  type,
+  setSelectedType,
+  isLoading,
+  isWhitelisted,
+  setApplyModalEnabled,
 }: TierProps) {
   const address = useAddress()
   const { login, user, logout } = usePrivy()
 
-  // const { login } = useLogin({
-  //   onComplete: (user, isNewUser, wasAlreadyAuthenticated) => {
```

Comment:
Get rid of commented out code

Reply:
The '-' next to the code means it's been removed.